languages/python/jupyter/sbatch_jupyter.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash  --login
+
+#SBATCH --time=01:00:00
+#SBATCH --nodes=1
+#SBATCH --partition=short
+#SBATCH --tasks-per-node=1
+#SBATCH --cpus-per-task=36
+#SBATCH --account=<account-handle>
+
+module purge
+module load conda
+conda activate /home/$USER/.conda/envs/jupyterenv
```

Comment:
safer option for eagle users is usually source activate:
```suggestion
source activate /home/$USER/.conda/envs/jupyterenv
```